app/views/items/index.html.erb

```diff
@@ -4,6 +4,19 @@
 <% else %>
   <h1>All Items</h1>
 <% end %>
+
+<section id="statistics">
+  <section id="most-popular">
+    <h3><b> Most Popular Sold Items</b></h3>
+
```

Comment:
Well done here. This'll look great on the page. Solid indentation as well keeping all the HTML organized an readable.